### PR TITLE
Feedback and fixes for Python course

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Hi there, excellent course, of course! I could only spot trivial things while "browsing" the chapter on Python. I hope you find this feedback useful.

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
For your review @vbellee and @chrisburr.

**Comment by apuignav** [comment]
Looks good to me! The tests seem to be failing due to some problem loading a plugin.

@chrisburr do you know how to fix it? Travis is well beyond my expertise...

File: python/methods.md [hunk 1]
```diff
@@ -106,7 +107,9 @@ Methods can be called in several ways.
 >>> add(1, y=2)
 >>> add(y=2, x=1)
 >>> add(y=2, 1)
->>> add(y=2, 1)
+  File "<stdin>", line 1
+SyntaxError: non-keyword arg after keyword arg
+>>> add(y=2, =1)
```
Comment (alexpearce) [inline python/methods.md:112]: There's a syntax error here, `, =1`.

Comment (eduardo-rodrigues) [reply python/methods.md:112]: That was already in the course, on purpose I guess. But there were some mistakes too, including a stack trace that did not correspond. I fixed both things - no problem in showing a mistake here and there, I reckon.

**Comment by eduardo-rodrigues** [comment]
Hi, I'm done with my feedback from having a quick-ish read through the course. I did not try and expand sections such as the final one. That's for the future, maybe.

I hope this PR can be reviewed and accepted soon. Thanks for having a look.

File: python/methods.md (hunk 1 above)
Comment (eduardo-rodrigues) [reply python/methods.md:112]: To be clear: now the examples are either correct or bad usecases and the latter show a corresponding stack trace.

**Comment by chrisburr** [comment]
I've fixed travis (we needed to use a newer version of node due to lhcb/gitbook-plugin-panels/pull/3).

I'm happy with these changes so I'll merge this later today if no one objects or leaves further comments.

**Comment by vbellee** [comment]
I'm happy as well with the changes.

**Comment by apuignav** [comment]
LGTM!

**Comment by chrisburr** [comment]
LGTM :+1: Thanks for taking the time to make these improvements @eduardo-rodrigues.

**Pull request merged**